onnxconverter_common/optimizer.py

```diff
@@ -1100,7 +1118,7 @@ def _process_transpose_pass_broadcast(node, node_list, node_transpose_pass_name,
     return node_list, cur_perm_map
 
 
-def _process_transpose_pad(node, node_list, node_transpose_pass_name, cur_perm_map):
+def _process_transpose_pad(node, node_list, node_transpose_pass_name, cur_perm_map, target_opset):
```

Comment:
should we put opset into node to avoid so many changes on the function argument?

Reply:
done